composer.json

```diff
@@ -16,7 +16,7 @@
     "license" : "MIT",
     "require" : {
         "masterminds/html5" : "^2.1.2",
-        "twig/twig" : "^1.10||^2.0",
+        "twig/twig" : "^1.28||^2.0",
```

Comment:
I see... :(

can you update also `.travis.yml` updating the environment variable ?